docs/modules/ROOT/pages/how-to/use-k8up.adoc

```diff
@@ -0,0 +1,351 @@
+= How to Backup with K8up
+
+image:logos/k8up.svg[role="related thumb right",alt="K8up logo",width=120,height=120] https://k8up.io/[K8up] (pronounced /keɪtæpp/ or simply "ketchup") is a Kubernetes Operator distributed via a Helm chart, compatible with OpenShift and plain Kubernetes, and installed by default in {product}
+
+This page will describe the required steps to use K8up successfully with {product}.
+
+IMPORTANT: This guide is provided as a courtesy for {product} users. For more information about K8up, please refer to its https://k8up.io/[official documentation].
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tool installed:
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry.
+
+`restic`:: Follow the https://restic.net/#installation[Restic installation instructions] corresponding to your operating system.
+
+== Procedure
+
+This section explains the steps required to use K8up with {product}.
+
+=== 1. Login to {product}
+
+include::partial$login-in-terminal.adoc[]
+
+. Create a new project called "[YOUR_USERNAME]-fortune-go"
++
+[source,shell]
+--
+oc new-project [YOUR_USERNAME]-fortune-go
+--
+
+=== 2. Deploy a simple app
+
+We're going to deploy a https://gitlab.com/vshn/applications/fortune-go[sample application written in Go] using the `deployment.yaml` file shown below. This deployment doesn't need neither a `Service` nor an `Ingress`:
+
+[source,yaml]
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fortune-go
+spec:
+  template:
+    spec:
+      containers:
+      - image: registry.gitlab.com/vshn/applications/fortune-go:latest
+        imagePullPolicy: Always
+        name: fortune-container
+        ports:
+        - containerPort: 8080
+    metadata:
+      labels:
+        app: fortune-go
+      annotations:
+        k8up.io/backupcommand: fortune # <1>
+        k8up.io/file-extension: .txt   # <2>
+  selector:
+    matchLabels:
+      app: fortune-go
+  strategy:
+    type: Recreate
+--
+<1> K8up is able to backup the output of any tool piping data to `stdin`; in this case, we're going to backup a message created by the `fortune` tool installed in the pod.
+<2> This annotation provides a file extension to the data to be added in the backup.
+
+Apply the deployment to your {product} project:
+
+[source,shell]
+--
+oc apply -f deployment.yaml
+--
+
+=== 3. Deploy Minio
+
+K8up is powered by the https://restic.net/[Restic] backup tool, and is thus able to store backups in any S3-compatible storage. For the purposes of this guide, you can quickly install a https://min.io/[MinIO] instance in your project.
```

Comment:
Maybe add a note that for production it's better to use an S3 endpoint outside of the cluster.